### DISCUSSION TITLE
Asset field view mode customisable per layout

### DISCUSSION THREAD

**Discussion by thupsi** [craftcms/cms#11392] [Ideas] [answered]
A small thing: It would make sense to allow changing the ``view mode`` of asset fields in the field layout editor _per layout_, in the same way that we currently can change the ``label`` for every field. 

I am aware that the plan is to refactor fields in a way that they are more easily reusable across sections and inside matrix, but in the meantime maybe this small detail would be easy to implement.

**Comment by brandonkelly** [comment, marked as answer]
This is basically the same request as #8688. If we do end up implementing it, Assets fields’ View Mode settings would be a good example of what we could do with it. But currently it’s not planned (see https://github.com/craftcms/cms/discussions/8688#discussioncomment-2811602).

**Reply by thupsi** [reply]
Yeah, I've seen that discussion. Still, view mode only affects the appearance in the CP, so I thought it would be safer/simpler to tamper with.

**Reply by brandonkelly** [reply]
Field layout designers don’t have any knowledge of field type-specific settings, and we wouldn’t go and add a one-off exception for one particular Assets field setting. It would need to be implemented as some sort of underlying framework for field types to get involved in field layout designer settings across the board, and then we could take advantage of that from `craft\fields\Assets` from there.

**Reply by thupsi** [reply]
That makes sense. Thanks for taking the time to explain!
